.github/pie-docs-rewrites.txt

```diff
@@ -0,0 +1,32 @@
+[
+    {
```

Comment:
I'd be tempted to have a strict 1:1 naming convention here with the options in the UI

Reply:
This is copied directly from the editor. I thought it would be the most simple option